tools/labs/templates/deferred_work/1-2-timer/timer.c

```diff
@@ -17,7 +17,7 @@ MODULE_LICENSE("GPL");
 
 static struct timer_list timer;
 
-static void timer_handler(unsigned long var)
+static void timer_handler(struct timer_list *timer)
```

Comment:
MAYBE: Use a different name for the argument or for the global variable, this way you shadow the global variable.